operatorconfig/moduleconfig/authorization/v1.6.0/policies.yaml

```diff
@@ -312,7 +178,7 @@ data:
       common.roles[a]
 
       v := claimed_roles[i]
-      common.roles[v].system_types[input.systemtype].system_ids[input.storagesystemid].pool_quotas[input.storagepool] >= to_number(input.request.volumeSizeInKb)
+      common.roles[v].system_types[input.systemtype].system_ids[input.storagesystemid].pool_quotas[input.storagepool] == 0
```

Comment:
Can we add this change to the volumes-create policy above?

Reply:
fixed